webapp/package.json

```diff
@@ -21,7 +21,7 @@
     "@babel/plugin-syntax-dynamic-import": "^7.2.0",
     "@babel/preset-env": "^7.4.5",
     "@babel/preset-react": "^7.0.0",
-    "@geppettoengine/geppetto-client": "openworm/geppetto-client#2.0.0-alpha",
+    "@geppettoengine/geppetto-client": "git://github.com:openworm/geppetto-client.git#2.0.0-alpha-branch-all-plots",
```

Comment:
If it's not really needed, better stick with 2.0.0

Reply:
We need this one unfortunately since the ListViewer component needs a one line change to allow for a component to be displayed between the search filter and the table

Reply:
There is the new `layout` prop for this: 
```
<ListViewer
                    ...
          layout={({ Table, Pagination, Filter, SettingsWrapper }) => (...)}
 />
```

Reply:
This is available on the last development branch